migrations/1757326540_settings.go

```
@@ -17,6 +17,21 @@ func init() {
+			// Backups.S3 for that. Env-overridable; only set when unconfigured
+			// so we never clobber a schedule set via the admin UI.
+			if settings.Backups.Cron == "" {
+				cron := os.Getenv("PRIMO_BACKUP_CRON")
+				if cron == "" {
+					cron = "0 3 * * *" // daily at 03:00
+				}
+				settings.Backups.Cron = cron
+				settings.Backups.CronMaxKeep = 7
+			}
+
 			app.Save(settings)
```

golangci-lint (2.12.2)

[error] Error return value of `app.Save` is not checked

(errcheck)